common/meta/grpc/src/grpc_client.rs

```diff
@@ -346,7 +366,18 @@ impl MetaGrpcClient {
                 }
                 message::Request::MakeClient(_) => {
                     let resp = self.make_client().await;
-                    resp.map(message::Response::MakeClient)
+                    match resp {
+                        Ok(r) => Ok(message::Response::MakeClient(r)),
+                        Err(e) => {
+                            tracing::warn!("make client error, try to remake. {:?}", e);
+                            let res = self.make_client().await;
+                            res.map(message::Response::MakeClient)
+                        }
+                    }
+                }
```

Comment:
Isn't it better to let `self.make_client()` do the retry-loop when failed to build a client?

Reply:
Good idea